tower-grpc/src/body.rs

```diff
@@ -60,6 +60,11 @@ pub struct BoxBody {
 
 struct MapBody<B>(B);
 
+#[derive(Debug, Default)]
+struct EmptyBody {
```

Comment:
I don't think this is being used anywhere is it?

Reply:
Right it is no longer used; it's been removed!